Lesson3/Dariia-Skoryk/task4_stack/task/main.h

```diff
@@ -0,0 +1,68 @@
+#pragma once
+
+#include <iostream>
+#include <stack>
+#include <string>
+#include <exception>
+
+bool isExpressionValid(const std::string & expression) {
+    //smth
+    bool result = true;
+    std::stack<char> openBrackets;
+    for (const char entry: expression) {
+        switch (entry) {
+            case '(':
```

Comment:
????
why not push entry?
in this case you can extract this check to some predicate function